app/src/main/java/com/github/sdp/mediato/utility/adapters/UserAdapter.java

```diff
@@ -57,11 +83,15 @@ public static class UserViewHolder extends RecyclerView.ViewHolder {
 
         ImageView userProfileImageView;
         TextView userNameTextView;
+        Button followButton;
+        Button unfollowButton;
 
         public UserViewHolder(@NonNull View itemView) {
             super(itemView);
-            userProfileImageView = itemView.findViewById(R.id.searchUserAdapter_imageView);
-            userNameTextView = itemView.findViewById(R.id.searchUserAdapter_userName);
+            userProfileImageView = itemView.findViewById(R.id.userAdapter_imageView);
+            userNameTextView = itemView.findViewById(R.id.userAdapter_userName);
+            followButton = itemView.findViewById(R.id.userAdapter_followButton);
+            unfollowButton = itemView.findViewById(R.id.userAdapter_unfollowButton);
```

Comment:
Bindings might make your life easier here.

Reply:
can you be more specific or show an example?